include/SFML/Graphics/Shape.hpp

```diff
@@ -52,6 +52,30 @@ class SFML_GRAPHICS_API Shape : public Drawable, public Transformable
     ////////////////////////////////////////////////////////////
     ~Shape() override;
```

Comment:
https://godbolt.org/z/cYs4exPs3

@kimci86 Brought up how we can apparently remove `~Shape` which will implicit define all these other special member functions. Because `sf::Shape` inherits from a class that has a virtual distractor, it too inherits a virtual destructor. The above Godbolt link shows that this won't introduce memory leaks. We can use `std::has_virtual_destructor_v<T>` to assert this fact and ensure regressions don't slip in.